packages/blade/src/tokens/global/fontFamily.native.ts

```diff
@@ -0,0 +1,6 @@
+const fontFamily = {
+  text: 'Lato',
+  code: 'monospace',
```

Comment:
Need to test. Will check and respond

Reply:
Any update?

Reply:
https://github.com/react-native-training/react-native-fonts


Checked a couple of places. Seems like iOS doesn't have `monospace` :(

Reply:
![Screenshot_1621838278](https://user-images.githubusercontent.com/11384858/119307523-725a6400-bc89-11eb-93e4-42ec81b1e83b.png)

Tested on android. iOS my simulator is not in a mood to run today and failing randomly. Since this is a rare use case of using `monospace` it's acting as a placeholder. Can we come back to this once we have a case for this?